Package.swift

```diff
@@ -11,18 +11,18 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/apple/swift-log.git", from: "1.0.0"),
-        .package(url: "https://gitlab.com/mordil/swift-redi-stack.git", from: "1.0.0-alpha.7"),
+        .package(name: "redi-stack", url: "https://gitlab.com/mordil/swift-redi-stack.git", from: "1.0.0-alpha.7"),
```

Comment:
I don't think this is required since it's declared in the packages manifest

Reply:
Without the build fails with actual 5.2 (only testing on Linux, not macOS).
```
'redis-kit' /home/a/Schreibtisch/redis-kit: error: product dependency 'RediStackTestUtils' in package 'swift-redi-stack' not found
'redis-kit' /home/a/Schreibtisch/redis-kit: error: product dependency 'RediStack' in package 'swift-redi-stack' not found
```
Not sure tho if this is the best solution

Reply:
You can see it here: https://github.com/saicu/redis/runs/493164967
Thats the run with the non-changed Package.swift

Reply:
Ok this is definitely weird. I think there's some undefined behaviour in SwiftPM's resolver where the name of the Package doesn't match up with the Git URL. The fact that this builds in Xcode 11.4 Beta 3 suggests that we should get that clarified before merging anything. Do you want to file an issue on bugs.swift.org?

Reply:
@0xTim yep, feels kinda fishy. I made a report here https://bugs.swift.org/browse/SR-12339

Reply:
@0xTim We got a reply there - my changes to the Package.swift are necessary